namerd/examples/k8s.yaml

```diff
@@ -0,0 +1,12 @@
+# This file assumes you are running through `kubectl proxy` on localhost:8001.
+storage:
+  kind: io.buoyant.namerd.storage.experimental.k8s
```

Comment:
generally, i have mixed feelings on using class paths for kinds.  I know we've done this for other ones, but it encodes a bunch of redundant information that isn't particularly user-friendly.  We can revisit this outside of this review, though.